grpc/src/main/java/com/linecorp/armeria/server/grpc/StreamingServerCall.java

```diff
@@ -224,7 +225,7 @@ void doClose(Status status, Metadata metadata, boolean completed) {
     }
 
     @Override
-    @Nullable O firstResponse() {
+    public @Nullable O firstResponse() {
```

Comment:
```suggestion
    protected @Nullable O firstResponse() {
```

Reply:
Done in [63e0b79](https://github.com/line/armeria/pull/4724/commits/63e0b79cdf01d14c58d916c29c87d1f129f255a8)!